artifacts/pom.xml

```diff
@@ -135,16 +135,40 @@
 		<!-- latest mock-sdk -->
 		<mock-sdk-latest.version>1.2.0.2</mock-sdk-latest.version>
 
+		<!-- certify sunbird plugin -->
 		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
-		<!-- certify sunbird plugin  -->
-		<certify-sunbird-plugin.version>0.2.1</certify-sunbird-plugin.version>
+		<certify-sunbird-plugin.version>0.3.0-SNAPSHOT</certify-sunbird-plugin.version>
 		<certify-sunbird-plugin.fileName>certify-sunbird-plugin.jar</certify-sunbird-plugin.fileName>
 		<!-- certify mosip identity plugin -->
-		<certify-mosip-identity-plugin.version>0.2.1</certify-mosip-identity-plugin.version>
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mosip-identity-plugin.version>0.3.0-SNAPSHOT</certify-mosip-identity-plugin.version>
 		<certify-mosip-identity-plugin.fileName>certify-mosip-identity-plugin.jar</certify-mosip-identity-plugin.fileName>
-		<!-- certify mock plugin -->
-		<certify-mock-plugin.version>0.2.1</certify-mock-plugin.version>
+        <!-- certify mock plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mock-plugin.version>0.3.0-SNAPSHOT</certify-mock-plugin.version>
 		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
+
+		<!-- certify mock ida dataprovider plugin -->
+		<certify-mock-ida-dataprovider-plugin.version>0.3.0-SNAPSHOT</certify-mock-ida-dataprovider-plugin.version>
+        <certify-mock-ida-dataprovider-plugin.fileName>certify-mock-ida-dataprovider-plugin.jar</certify-mock-ida-dataprovider-plugin.fileName>
```

Comment:
why we this mock ida dataprovider plugin, it is suppose to be inside the mock jar file.

Reply:
Removed mock-ida-dp plugin as this will be integrated in mock-certify-plugin project. A ticket has been created for the same: https://mosip.atlassian.net/browse/INJICERT-643